lib/cli/src/commands/init.rs

```diff
@@ -0,0 +1,472 @@
+                    })
+                    .collect(),
+            ),
+            BinOrLib::Lib | BinOrLib::Empty => None,
+        }
+    }
+
+    /// Returns the dependencies based on the `--template` flag
+    fn get_dependencies(&self) -> Option<HashMap<String, String>> {
+        Some({
+            match self.template.as_deref() {
+                Some("js") => {
+                    let mut map = HashMap::default();
+                    map.insert("python".to_string(), "quickjs/quickjs@latest".to_string());
+                    map
+                }
+                Some("python") => {
+                    let mut map = HashMap::default();
+                    map.insert("python".to_string(), "python/python@latest".to_string());
+                    map
+                }
+                _ => HashMap::default(),
+            }
+        })
+    }
+
+    // Returns whether the template for the wapm.toml should be a binary, a library or an empty file
+    fn get_bin_or_lib(&self) -> Result<BinOrLib, anyhow::Error> {
+        match (self.empty, self.bin, self.lib) {
+            (true, true, _) | (true, _, true) => Err(anyhow::anyhow!(
+                "cannot combine --empty with --bin or --lib"
+            )),
+            (true, false, false) => Ok(BinOrLib::Empty),
+            (_, true, true) => Err(anyhow::anyhow!(
+                "cannot initialize a wapm manifest with both --bin and --lib, pick one"
+            )),
+            (false, true, _) => Ok(BinOrLib::Bin),
+            (false, _, true) => Ok(BinOrLib::Lib),
+            _ => Ok(BinOrLib::Bin),
+        }
+    }
+
+    fn get_bindings(target_file: &Path, bin_or_lib: BinOrLib) -> Option<wapm_toml::Bindings> {
+        match bin_or_lib {
+            BinOrLib::Bin | BinOrLib::Empty => None,
+            BinOrLib::Lib => target_file.parent().and_then(|parent| {
+                walkdir::WalkDir::new(parent)
+                    .min_depth(1)
+                    .max_depth(3)
+                    .follow_links(false)
+                    .into_iter()
+                    .filter_map(|e| e.ok())
+                    .filter_map(|e| {
+                        let is_wit = e.path().extension().and_then(|s| s.to_str()) == Some(".wit");
+                        let is_wai = e.path().extension().and_then(|s| s.to_str()) == Some(".wai");
+                        if is_wit {
+                            Some(wapm_toml::Bindings::Wit(wapm_toml::WitBindings {
+                                wit_exports: e.path().to_path_buf(),
+                                wit_bindgen: semver::Version::parse("0.1.0").unwrap(),
+                            }))
+                        } else if is_wai {
+                            Some(wapm_toml::Bindings::Wai(wapm_toml::WaiBindings {
+                                exports: None,
+                                imports: vec![e.path().to_path_buf()],
+                                wai_version: semver::Version::parse("0.1.0").unwrap(),
```

Comment:
WAI is currently at version `0.2.0` (technically `0.2.2`, but this will match any `0.2.x`).

```suggestion
                                wai_version: semver::Version::parse("0.2.0").unwrap(),
```